aws/resource_aws_config_configuration_aggregator_test.go

```diff
@@ -58,8 +60,8 @@ func testSweepConfigConfigurationAggregators(region string) error {
 
 func TestAccAWSConfigConfigurationAggregator_account(t *testing.T) {
 	var ca configservice.ConfigurationAggregator
-	rName := acctest.RandomWithPrefix("tf-acc-test")
-	resourceName := "aws_config_configuration_aggregator.example"
+	rName := acctest.RandomWithPrefix("Tf-acc-test")
```

Comment:
Add a comment that an uppercase character in the resource name is explicitly being tests for so that this isn't inadvertently changed in the future.

Reply:
Thanks, added.